apps/coordinator/.env.example

```
@@ -1,4 +1,6 @@
 HTTP_SERVER_PORT=8020
 PLATFORM_ENABLED=true
 PLATFORM_WS_PORT=3030
-SECURE_CONNECTION=false
\ No newline at end of file
+SECURE_CONNECTION=false
+# Dev-only: stream this process's logs over a local telnet/TCP socket (nc localhost 6770). Unset to disable.
+COORDINATOR_TELNET_LOGS_PORT=6770
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)